website/templates/project/registrations.mako

```diff
@@ -0,0 +1,161 @@
+                                  click: $root.editDraft">
+                      <i style="margin-right: 5px;" class="fa fa-pencil"></i>Edit
+                    </a>
+                    <a class="btn btn-info"
+                       data-bind="visible: isPendingApproval,
+                                  click: $root.previewDraft">
+                      <i style="margin-right: 5px;" class="fa fa-pencil"></i>Preview
+                    </a>
+                    <button class="btn btn-danger"
+                            data-bind="click: $root.deleteDraft.bind($root)">
+                      <i style="margin-right: 5px;" class="fa fa-times"></i>Delete
+                    </button>
+                  </div>
+                  <div class="col-md-1">
+                    <span data-bind="ifnot: requiresApproval">
+                     <a class="btn btn-success" data-bind="attr: {href: urls.register_page}">Register</a>
+                    </span>
+                  </div>
+                </div>
+              </h4>
+            </li>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+</div>
+<script type="text/html" id="createDraftRegistrationModal">
+    <p>Registration creates a frozen version of the project. Your original project remains editable and will have the registration linked. Things to know about registration:</p>
+    <ul>
+        <li><strong>Registrations cannot be edited or deleted.</strong></li>
+        <li>Withdrawing a registration removes its contents, but leaves behind basic metadata: title, contributors, date registered, date withdrawn, and justification (if provided).</li>
+        <li>Registrations can be public or embargoed for up to four years. Embargoed registrations will be made public automatically when the embargo expires.</li>
+    </ul>
+
+    <p>Continue your registration by selecting a registration form:</p>
+    <span data-bind="foreach: schemas">
+    <div class="radio">
+        <label>
+          <input type="radio" name="selectedDraftSchema"
+                 data-bind="attr: {value: id}, checked: $root.selectedSchemaId" />
+          <span data-bind="text: schema.title"></span>
+          <!-- ko if: schema.description -->
+          <i data-bind="tooltip: {title: schema.description}" class="fa fa-info-circle"> </i>
+          <!-- /ko -->
+        </label>
+    </div>
+    </span>
+</script>
+<%def name="javascript_bottom()">
+    ${parent.javascript_bottom()}
+    <script>
+        window.contextVars.analyticsMeta = $.extend(true, {}, window.contextVars.analyticsMeta, {
+            pageMeta: {
+                title: 'Registrations',
+                public: true,
+            },
+        });
+    </script>
+
+    <script src=${"/static/public/js/project-registrations-page.js" | webpack_asset}> </script>
+</%def>
+
+<%include file="project/registration_preview.mako" />
+<%include file="project/registration_utils.mako" />
```

Comment:
@Johnetordoff , I think this is a bad merge, it doesn't look like website/templates/project/registrations.mako exists anymore, after the page was emberized.  So delete the file?

Reply:
I was confused by this because it appeared as a merge conflict, but I see now.